macos/FluentUI/AvatarView/AvatarView.swift

```diff
@@ -415,6 +415,7 @@ open class AvatarView: NSView {
 
 	open override func viewWillMove(toWindow newWindow: NSWindow?) {
 		super.viewWillMove(toWindow: newWindow)
+		updateAppearance(newWindow?.effectiveAppearance)
```

Comment:
this was missed in last PR, caught it while testing, just piggybacking changes in this PR